modules/ROOT/pages/basketball-robot-guide/mechanics/milling.adoc

```diff
@@ -0,0 +1,430 @@
+
+image::mill_stepcraft_xyz.png[]
+
+The machine's XYZ-axis directions are shown in the image.
+The origin point of the machine is also near here.
+When homing, the machine moves the spindle to the end of the Y-axis.
+This is done so there is better access to the table of the machine.
+
+
+
+== Before milling
+
+=== 1. Inspect the machine
+
+Check if any of the rail covers are open, if there are any material pieces on the table,
+if everything looks the way it should.
+
+
+=== 2. Clean the table to remove debris
+
+You can use a handbrush or the vacuum to clean the table.
+
+
+=== 3. Disengage the Emergency Stop button
+
+The mill has no power when the Emergency Stop button is engaged. +
+The button can be disengaged by twisting it clockwise until it pops out.
+
+
+=== 4. Launch UCCNC
+
+
+=== 5. Disable Reset in UCCNC
+
+
+=== 6. Perform Double-Homing action
+
+This will move the mill all the way to the end of each axis, calibrating the position of the mill and
+giving you room to place the workpiece on the table.
+
+
+=== 7. Make sure that the mill can move around in XYZ-axis
+
+The mill should be freely movable around the workarea.
+
+
+=== 8. Clean up your material
+
+Remove plastic coverings off your material and brush over your material to remove debris.
+
+
+=== 9. Place pieces of double-sided tape on the bottom of the material
+
+Do not remove the coverings off the double-sided tape pieces.
+The tape should be attached to the extremities (corners and edges)
+and in the region where your parts will be milled out.
+Empty regions should also have a few pieces of tape.
+When placing the tape, make sure that the corners of the pieces don't go under themselves.
+
+
+=== 10. Remove the coverings off the double-sided tape
+
+Lay your material on a flat surface. Remove the coverings off the tape pieces.
+
+It's better to work your way from the back to the front when removing the covering off the tape pieces.
+
+
+=== 11. Place your workpiece on the table
+
+Make sure to properly align the piece.
+The workpiece must not hang over the table and should be on the spoilboard.
+
+A line is drawn on the spoilboard which can be used to align your material.
+
+
+=== 12. Insert your first tool into the spindle
+
+image::mill_collet.png[]
+
+* Check that the collet is properly attached to the clamping nut.
+** If the collet is loose, then it should be attached, without a tool inserted, into the clamping nut.
+An audible click should be heard.
+
+* The tool should be in the collet as deep as possible,
+but such that the collet is not covering the cutting edges of the tool.
+** Leaving 1~2 mm between the cutting edge and the collet is enough.
+
+* When attaching the tool, the locking button of the spindle has to be pressed in.
+* Screw the clamping nut onto the spindle as much as you can with fingers, then use a 17 mm wrench. +
+* Do not attach the clamping nut very tightly - when you later have to remove the collet, using too much force

```

Comment:
Would word as moderate force. It can not be too loose or the tool might come undone. I would compare it so that the machine splindle should be moved by the fastening force when fastening the collet.